README.md

```diff
@@ -554,8 +554,14 @@ also be read using `ssb.db.onMsgAdded.value`.
 
 ### getStatus
 
-Gets the current db status, same functionality as
-[db.status](https://github.com/ssbc/ssb-db#dbstatus) in ssb-db.
+Gets the current db status, same functionality (observable) as
+[db.status](https://github.com/ssbc/ssb-db#dbstatus) in ssb-db. The
+value contains the following information:
+
+- `log`: offset in log
```

Comment:
This is the standard terminology we use in all db2 related modules: see https://github.com/ssbc/async-append-only-log/ README. 

```
This module is not compatible with flume without a wrapper around stream as it uses the same terminology as [JITDB](https://github.com/ssb-ngi-pointer/jitdb/) and [ssb-db2](https://github.com/ssb-ngi-pointer/ssb-db2/) of using **offset** for the byte position of a record instead of seq.
```

We made sure that this was consistent across all modules :-) Before these things could be very confusing with seq and sequence and what not :)

Reply:
crap, format not working in comments...

Reply:
also what is the log?

```suggestion
- `log`: current offset of the head of the log in bytes (the log is the primary source of truth where raw messages are stored)
```